microshift_getting_started/microshift-architecture.adoc

```diff
@@ -2,30 +2,63 @@
 [id="microshift-architecture"]
 = Architecture
 include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
 :context: microshift-architecture
 
 toc::[]
 
-Learn the specifics of {product-title} architecture including design intent, how it differs from {OCP}, and API compatibility.
+Learn the specifics of {product-title} architecture including design intent, how it differs from {oke}, and API compatibility.
 
 [id="microshift-architectural-design_{context}"]
 == Architectural design
-{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
+{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. Because {product-title} is primarily as a platform for deploying applications, only the APIs and features essential to operating in edge and small form factor computing environments are included.
+
+For example, {product-title} is tasked with only the following Kubernetes cluster capabilities:
+
+* Networking
+* Ingress
+* Storage
+* Helm
+
+{product-title} also provides the following Kubernetes functions:
+
+* Orchestration
+* Security
+
+{product-title} is optimized when used with a compatible operating system, such as {op-system-ostree-first}. When used together, {product-title} and {op-system-ostree-first} form {op-system-bundle}. Virtual machines are handled by the operating system in {product-title} deployments.
+
+.{product-title} as part of {op-system-bundle}.
+image::311_RHDevice_Edge_Overview_0223_1.png[<{product-title} is tasked with only the Kubernetes cluster services networking, ingress, storage, helm, with additional Kubernetes functions of orchestration and security.>]
```

Comment:
I think this alt text could be improved to be more descriptive of the image. Something like "A multi-box diagram of Red Hat Device Edge with different boxes corresponding to different parts of the architecture. etc etc". This diagram is pretty complex, and i'm not sure what the user is supposed to focus on here. If it's the MicroShift component, the alt text could call out that box specifically and describe what it contains

Here are some links that were given to me about this, passing them on! 😄 

- https://www.w3.org/WAI/tutorials/images/complex/
- https://sc.edu/about/offices_and_divisions/digital-accessibility/guides_tutorials/alternative_text/charts-diagrams/index.php ("Example of Describing a Diagram")

Reply:
also, what are the `<` and `>` for?

Reply:
I think the `<` and `>` place the caption? Just picked it up from the repo.

Reply:
The alt text is part of the file, not the caption. It should be there, but I can't see it now that I'm using the ones from graphics.